.env

```
@@ -123,4 +123,6 @@ RECREATE_QA_USERS=false
 FILESYSTEM_ROSTERING_FILES_ADAPTER_S3_PREFIX=
 ALLOWED_UPLOADED_FILE_MAX_SIZE=5242880
 ALLOWED_UPLOADED_FILE_MAX_RECORDS=10000
+ROSTERING_PRINCIPALS_PORTAL_ENABLED=false
+ROSTERING_PRINCIPALS_PORTAL_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ROSTERING_PRINCIPALS_PORTAL_BASE_URL key should go before the ROSTERING_PRINCIPALS_PORTAL_ENABLED key

(UnorderedKey)